addons/base_import_module/models/ir_module.py

```diff
@@ -194,6 +194,16 @@ def _import_module(self, module, path, force=False, with_demo=False):
             'res_id': asset.id,
         } for asset in created_assets])
 
+        translation_importer = TranslationImporter(self.env.cr, verbose=False)
+        for lang_ in self.env['res.lang'].get_installed():
+            lang = lang_[0]
+            po_paths = get_po_paths(module, lang, env=self.env)
+            for po_path in po_paths:
+                translation_importer.load_file(po_path, lang)
+            if lang != 'en_US' and not po_paths:
+                _logger.info('module %s: no translation for language %s', module, lang)
```

Comment:
afther odoo/odoo#135607, the `get_resource_path` is replaced by `join` the po_paths won't be empty anymore I think. So you cannot check `no translation for language`. Maybe refactor `def _load_module_terms(`  as well as 
```py
po_paths = [
        path
        for filename in po_names
        for dir_ in ('i18n', 'i18n_extra')
        if (path := join(module_name, dir_, filename + '.po'))
     ]
```
in `get_po_paths`